plaso/parsers/sqlite_plugins/dropbox.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+"""SQLite parser plugin for Dropbox sync_history database files."""
+
+from dfdatetime import posix_time as dfdatetime_posix_time
+
+from plaso.containers import events
+from plaso.containers import time_events
+from plaso.lib import definitions
+from plaso.parsers import sqlite
+from plaso.parsers.sqlite_plugins import interface
+
+
+class DropboxSyncHistoryEventData(events.EventData):
+  """Dropbox Sync History Database event data.
+
+  Attributes:
+    event_type (str): the event type
+    file_event_type (str): the file event type
+    direction (str): the source of the synchronisation event
+    file_id (str): the ID of the file.
```

Comment:
id => identifier

Also what does this identifier contain? is this a dropbox identifier?

Reply:
Yes, this appears to be a Dropbox generated value to uniquely reference a file.

```
File id (e.g. id:odTlUvbpIEFAAAAAAAAGOQ)
    - Unique file reference that persists if files are moved or renamed
    - Useful for apps referencing files that are moved around
    - Best choice for reliable, accurate reference to a specific file
 ```
https://developers.dropbox.com/dbx-file-access-guide